Dispatchfile

```diff
@@ -10,10 +10,55 @@ load("github.com/mesosphere/dispatch-tasks/bump_charts/bump_charts@master", "bum
 git_clone_dirname = "kba-git-src"
 kind_cluster_name = "kba-kind-cluster"
 
+resource("artifacts", type="storage", params={
+    "type": "gcs",
+    "location": "s3://artifacts",
+    "dir": "yes"
+}, secrets={
+    "BOTO_CONFIG": k8s.corev1.SecretKeySelector(key="boto", localObjectReference=k8s.corev1.LocalObjectReference(name="s3-config"))
+})
+
 git_resource(git_clone_dirname, url="$(context.git.url)", revision="$(context.git.commit)")
+git_resource("ui-git", url="git@github.com:mesosphere/kommander.git", revision="v6.35.0")
```

Comment:
how does this get updated? manually by UI team?

Reply:
yes, when a new UI version is bumped or when a new test is merged we need to bump this